content/en/install.md

```diff
@@ -3,28 +3,38 @@ title: Installation
 sidebar: false
 ---
 
-Installations methods include:
+Installation methods include:
 
-- [Distributions](#distributions)
-- [pip](#pip-install)
-- [conda](#pip-install)
-- [Package Manager](#package_manager)
+- [Python Distributions](#distributions)
+- [PyPI (`pip`, `uv`)](#pypi-install)
+- [conda-forge (`conda`, `mamba`, `pixi`)](#conda-forge-install)
+- [System Package Managers](#system-package-manager)
 - [Source](#source)
 
 Methods differ in ease of use, coverage, maintenance of old versions,
-system-wide versus local environment use, and control. With pip or
-Anaconda\'s conda, you can control the package versions for a specific
+system-wide versus local environment use, and control. When installing
+from PyPI or with Conda, you can control the package versions for a specific
 project to prevent conflicts. Conda also controls non-Python packages,
 like MKL or HDF5. System package managers, like `apt-get`, install
-across the entire computer, often have older versions, and don\'t have
+across the entire computer, often have older versions, and don't have
 as many available versions. Source compilation is much more difficult
-but is necessary for debugging and development. If you don\'t know which
+but is necessary for debugging and development. If you don't know which
 installation method you need or prefer, we recommend the Scientific
 Python Distribution [Anaconda](https://www.anaconda.com/download/).
 
+Users in large, non-university institutions may want to read Anaconda's
```

Comment:
I won't block, but I disagree with putting this here. Users to which this really applies need to do such analysis anyway for all the tools they use, have procurement teams, etc.

This only sends a bad message for Anaconda which has been giving a lot (and still do) to this community.

Reply:
I think it is a useful pointer for us to include, and I don't think that linking out to Anaconda's own blog post sends a bad message (if anything, it could help make users who should be paying Anaconda aware that they need to!)

I think there were valid concerns raised in  https://docs.google.com/document/d/1zfncPEtNgI86pm8AhJ50DCGSTexcEIuA6jUofy2yqvU/edit?tab=t.0#heading=h.exik8qo8laee about a situation which was pretty confusing (at least for me) to understand, and Anaconda's blog post helpfully explained it.

But I also won't block, if folks would rather not include this :)

Reply:
I remember being against that doc and NF having an opinion 😉 This is political at best, again users to which this applies know what to do. There won't be many large orgs which don't have a proper process around compliant tooling. People who don't pay do it on purpose and this note won't help.

Also from a practical point of view, such link tend to break as time pass. And the TOC can always change.

Reply:
I think we should just change the recommendations to only use Conda-forge. There is no reason to use `defaults` at this point if you're an individual user trying to figure this stuff out. So we get rid of https://www.anaconda.com/download/ and then we don't need to worry about the ToS drama.

Reply:
> So we get rid of https://www.anaconda.com/download/

Are you suggesting removing the whole "Scientific Python Distributions" section?

Reply:
Yes